doc/release-notes/0.44/0.44.md

```diff
@@ -173,9 +173,9 @@ on x64 platforms, but builds with OpenJDK 17 and later also require more stack s
 
 <tr>
 <td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/60">#60</a></td>
-<td valign="top">The <tt>-XX:+CompactStrings</tt> option is having an adverse impact on the <tt>indexOf()</tt> method results.</td>
-<td valign="top">All platforms</td>
-<td valign="top"><tt>String</tt> compression is enabled by default because of which the non-Latin-1 characters are not identified. If you use the <tt>split()</tt> method to create a <tt>string</tt> from a <tt>string</tt> that contains non-Latin-1 characters and you use that new <tt>string</tt> with the <tt>indexOf()</tt> method, the results might be wrong.</td>
+<td valign="top">The <tt>-XX:+CompactStrings</tt> option is having an adverse impact on the OpenJDK 11 <tt>indexOf()</tt> method results.</td>
+<td valign="top">OpenJDK 11</td>
```

Comment:
The column heading should probably change from "Platform" to "Version / Platform" like the previous table.

Reply:
Updated along with more updates.